core/src/main/java/org/apache/iceberg/BaseMetastoreCatalog.java

```diff
@@ -301,4 +305,16 @@ protected static String fullTableName(String catalogName, TableIdentifier identi
 
     return sb.toString();
   }
+
+  private MetricsReporter metricsReporter() {
+    if (metricsReporter == null) {
+      metricsReporter =
+          properties().containsKey(CatalogProperties.METRICS_REPORTER_IMPL)
+              ? CatalogUtil.loadMetricsReporter(
+                  properties().get(CatalogProperties.METRICS_REPORTER_IMPL))
+              : LoggingMetricsReporter.instance();
```

Comment:
Looks like this will enable LoggingMetricsReporter for all catalogs now, no way to turn it off.  @kmozaid @nastra curious is it the desired behavior?

Reply:
HI @szehon-ho Even without this change, LoggingMetricsReporter is always enabled for all catalogs...See BaseTable https://github.com/apache/iceberg/blob/master/core/src/main/java/org/apache/iceberg/BaseTable.java#L45

Reply:
Makes sense, sorry I didnt have the context but wonder is there enough value to always fill the logs every time?  But agree its not part of this issue.